packages/api/prisma/schema.prisma

```diff
@@ -9,10 +9,21 @@ generator client {
 
 model User {
```

Comment:
Is this name correct? Maybe UserPersonalInfo or something like that will be more accurate?

Reply:
Great idea, but it's not in the scope of this PR